addons/spreadsheet/static/src/assets_backend/spreadsheet_binary_field/spreadsheet_binary_field.js

```diff
@@ -0,0 +1,30 @@
+import { registry } from "@web/core/registry";
```

Comment:
Why is the file in community if it's only used in an enterprise module ?

Reply:
I moved the field to the `spreadsheet_dashboard_view` in community